app/views/user_sessions/_form.html.erb

```diff
@@ -3,10 +3,10 @@
 <input type="hidden" name="return_to" class="login-modal-redirect" value="<%= request.fullpath %>" />
 
 <center>
-  <h2><%= raw t('user_sessions.new.log_in') %></h2>
+  <h2><%= raw translate('user_sessions.new.log_in') %></h2>
```

Comment:
Hi @stone95 , thanks for opening this :tada: this should be changed to "translation". Thanks

Reply:
Also please change the link on `Fixed #... ` to the link to the issue.